salt/_macros/certs.jinja

```diff
@@ -41,7 +41,7 @@
 
 #####################################################################
 
-{% macro certs(name, crt, key, cn='', o='', extra_alt_names=None) -%}
+{% macro certs(name, crt, key, cn='', o='', extra_alt_names=None, bundle=False) -%}
```

Comment:
It isn't clear offhand here that bundle is the full path to the bundle file. It is though, as it is used that way below. Perhaps 'bundle_file' would be more clear?

I do get though that crt and key are also full pathnames, so it does match the existing style for the macro.

Reply:
That consistency is is precisely why it's "bundle". :)